README.md

```diff
@@ -133,16 +111,13 @@ Users want to determine several types of the formatting width as following
   + microseconds
   + nanoseconds
 
-## Determining Time Fields
+### Determining Duration Units
+
+`DurationFormat` does not do any arithmetic operations on the input implicitly, expecting the user to use functions on the prototype like `Temporal.Duration.prototype.round` in order to ensure that the input is within a certain range if needed.
 
-* Users need to determine the time fields,
-* For example: 6059 seconds
-  + Hours, Minutes, Seconds
-    - 1 hours, 40 minutes and  59 seconds
-  + Minutes and Seconds
-    - 100 minutes and 59 seconds.
+To avoid accidentally truncating part of the value, `DurationFormat` doesn't allow hiding certain fields, so users expecting a specific result are highly recommended to call `round` in order to make sure that the input only includes necessary units.
```

Comment:
```suggestion
To avoid accidentally omitting part of the duration, `DurationFormat` always outputs all nonzero fields (except sub-second fields truncated by `fractionalDigits`).
Callers who want to omit nonzero fields (for example, only showing the date or time portion of the duration) should edit the input duration.
```